src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -2015,6 +2013,17 @@ protected void executeReport( Locale unusedLocale )
         }
         setFJavadocVersion( new File( jExecutable ) );
 
+        List<String> packageNames;

```

Comment:
I had to move these lines under `setFJavaVersion()` because `javadocRuntimeVersion` is not set before this point.